dashboard-app/src/app/theme/components/crop-image/ModalCtrl.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    'use strict';
+
+    angular.module('ConciergeApp.theme')
+        .controller('ModalCtrl', ModalCtrl);
+
+    function ModalCtrl ($scope,$timeout,$uibModalInstance) {
+        $scope.cropper = {};
+
+        $scope.getFileSystem = function () {
+           $timeout (function () {
+               var fileInput = document.getElementById('upload-image');
+               fileInput.click();
+           });
```

Comment:
Do you need to use $timeout without delay? Maybe there is need for it if you waiting for other stuff to load but if it is not necessary you can leave it out. also it would be good if you can leave out this document.getElementById and use more angular approach.